--- work/Configure.orig	2017-08-27 06:16:01.229756175 -0500
+++ work/Configure	2017-08-27 06:16:17.439253093 -0500
@@ -23612,7 +23612,7 @@
 
 : add -D_FORTIFY_SOURCE if feasible and not already there
 case "$gccversion" in
-[45].*)	case "$optimize$ccflags" in
+[45678].*)	case "$optimize$ccflags" in
 	*-O*)	case "$ccflags$cppsymbols" in
 		*_FORTIFY_SOURCE=*) # Don't add it again.
 			echo "You seem to have -D_FORTIFY_SOURCE already, not adding it." >&4
